src/libfetchers/git.cc

```diff
@@ -192,7 +192,10 @@ struct GitInputScheme : InputScheme
                 input.attrs.insert_or_assign("revCount", getIntAttr(infoAttrs, "revCount"));
             input.attrs.insert_or_assign("lastModified", getIntAttr(infoAttrs, "lastModified"));
             return {
-                Tree(store->toRealPath(storePath), std::move(storePath)),
+                Tree {
+                    store->toRealPath(store->makeFixedOutputPathFromCA(storePathDesc)),
```

Comment:
Could we add the field names to assignments like these? I am always a bit careful when most of the types are essentially strings and could be constructed from strings. You do not want the reordering of some fields in a struct to cause any side-effects on the actual code. Here at least `toRealPath` returns a string which is a valid input for the `StorePathDesc` type afaics.

Reply:
I would like to too!